### PR TITLE
Adding new translations to edunext-platform

### PR DESCRIPTION
**Pull request by vad33r**
Adding translations for the juniper-release + master of 2020-11-05 and Hindi

### CODE REVIEW

File: conf/locale/config.yaml [hunk 1]
```diff
@@ -46,46 +46,42 @@ locales:
...
+    # - id  # Indonesian
     - it_IT  # Italian (Italy)
     - ja_JP  # Japanese (Japan)
-    - ka  # Georgian
     # - kk_KZ  # Kazakh (Kazakhstan)
     # - km_KH  # Khmer (Cambodia)
     # - kn  # Kannada
     - ko_KR  # Korean (Korea)
-    - lt_LT  # Lithuanian (Lithuania)
-    - lv  # Latvian
```
Comment (felipemontoya) [inline conf/locale/config.yaml:58]: The standard we are using is to comment out the languages that we are not using. Not removing them from the list

File: conf/locale/config.yaml [hunk 2]
```diff
@@ -132,7 +128,7 @@ ignore_dirs:
     - src/done-xblock
     - src/edx-jsme
     - src/parse-rest
-    - src/geoip2
```
Comment (felipemontoya) [inline conf/locale/config.yaml:135]: This does not look to me like something we should be changing here. Is this an oversight or do we really need it?

File: conf/locale/config.yaml [hunk 3]
```diff
@@ -161,10 +157,6 @@ segment:
     djangojs-partial.po:
         djangojs-studio.po:
             - cms/*
-        djangojs-account-settings-view.po:
```
Comment (felipemontoya) [inline conf/locale/config.yaml:164]: This does not look to me like something we should be changing here. Is this an oversight or do we really need it?

File: conf/locale/config.yaml [hunk 4]
```diff
@@ -184,6 +176,5 @@ generate_merge:
     djangojs.po:
         - djangojs-partial.po
         - djangojs-studio.po
-        - djangojs-account-settings-view.po
```
Comment (felipemontoya) [inline conf/locale/config.yaml:187]: This does not look to me like something we should be changing here. Is this an oversight or do we really need it?

**Review by felipemontoya — CHANGES_REQUESTED** [review]
Some minor changes in the config.yaml. The rest looks good

File: conf/locale/config.yaml (hunk 1 above)
Comment (vad33r) [reply conf/locale/config.yaml:58]: Ok, i will fixed it and search for another deleted languages

File: conf/locale/config.yaml (hunk 2 above)
Comment (vad33r) [reply conf/locale/config.yaml:135]: I think this change is an oversight because of the compiled

**Review by felipemontoya — APPROVED** [review]
Good, this looks like it is ready to merge

**Pull request merged**